### PR TITLE
Add make dependency

### PR DESCRIPTION
**Pull request by marcelm**
#7 fails with `.../work/conda_build.sh: line 4: make: command not found`

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Closes #6
Closes #7

### CODE REVIEW

**Comment by marcelm** [comment]
@conda-forge-admin, please rerender

**Comment by marcelm** [comment]
@conda-forge-admin, please rerender

**Comment by marcelm** [comment]
@mfansler You described how to do a manual arch migration at https://github.com/conda-forge/pigz-feedstock/pull/18, and I tried to do the same here as well, could you have a look?

File: conda-forge.yml
Comment (mfansler) [inline conda-forge.yml:3]: Maybe also try adding the OSX ARM64 migration, while we're here:

```suggestion
build_platform:
  osx_arm64: osx_64
conda_build:
```

File: conda-forge.yml
Comment (mfansler) [inline conda-forge.yml:11]: Also part of the OSX ARM64 migration:

```suggestion
  linux_ppc64le: default
test: native_and_emulated
```

**Review by mfansler — APPROVED** [review]
LGTM!

You may want to: 

1. Also include the OSX ARM64 migration (see suggested code).
2. Add yourself as maintainer (if willing).

Both of those will need a rerender after. Also, you should add closing statements here so that the other PR(s) close once this merges.

Closes #6 #7 (<- I don't think me writing it triggers anything since I'm not a maintainer here).

**Comment by marcelm** [comment]
@conda-forge-admin, please rerender

**Review by martin-g — APPROVED** [review]

**Comment by mfansler** [comment]
@conda-forge/core as there is no `.github/CODEOWNERS` file here, I believe you'll need to merge this one. Thanks!

**Pull request merged**
